### PR TITLE
Add Carnegie Mellon University

### PR DESCRIPTION
**Pull request by Sunit-Kulkarni**
This is a redo of a previous pull request for this university. I fixed the merge conflicts.

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -43,6 +43,15 @@ websites:
       exceptions:
           text: "SMS-capable phone required for initial setup."
       doc: https://imss.caltech.edu/node/1408
+      
+    - name: Carnegie Mellon University
+      url: https://www.cmu.edu/
+      img: cmu.png
+      tfa: Yes
+      phone: Yes
```
Comment (mxxcon) [inline _data/education.yml]: I'm looking at their docs page and I don't see mention of phone calls. Is this field accurate?

Comment (Sunit-Kulkarni) [reply _data/education.yml]: Good catch. No I don't see phone either

**Review by mxxcon — DISMISSED** [review]

**Comment by brycx** [comment]
@Sunit-Kulkarni it seems to me you have removed CMU from the Purdue PR **and** this one. Is that on purpose? 😅

**Comment by Sunit-Kulkarni** [comment]
No I made a mistake. Forgot to git checkout the proper branch lol. Need to revert back to previous commit

**Comment by Carlgo11** [comment]
I cleaned up the PR.

**Review by Carlgo11 — DISMISSED** [review]

**Review by mxxcon — APPROVED** [review]

**Comment by Sunit-Kulkarni** [comment]
I fixed this in the MacStadium pull request.

**Review by mxxcon — DISMISSED** [review]
Could you please update the PR so that it's alphabetical?
Carnegie goes below CalTech. 👍

**Review by Carlgo11 — DISMISSED** [review]
How about this logo instead? It's their favicon and looks better on smaller scale (such as the 32x32 pixels that we use)
CMU

**Comment by Sunit-Kulkarni** [comment]
Good call. I like that logo better.

**Comment by Sunit-Kulkarni** [comment]
Also, is there a way to make the litany of commits not show up when a pull request is requested?

**Comment by mxxcon** [comment]
> Also, is there a way to make the litany of commits not show up when a pull request is requested?

You'd need to rebase your repo and then force-push...or something like that 🤔 
But when we merge site additions/changes we usually do "squash & merge" to collapse all these commits into a single one since in such cases intermediate commits/states are kinda irrelevant.
For a more complicated actual application code change perhaps it makes sense to keep intermediate commits to see the progress of that change.

**Comment by Carlgo11** [comment]
I replaced the old logo with the favicon one and somewhat cleaned up the PR in the process. Didn't want to do a complete rebase since that would mess up the timeline.

**Review by Carlgo11 — APPROVED** [review]

**Review by stephengroat — APPROVED** [review]

**Pull request merged**